.ruby-version

```diff
@@ -1 +0,0 @@
-2.1.1
```

Comment:
I think that if you want to remove the .ruby-version then that should be in a separate pull request, since it is unrelated to the main change being introduced here.